src/python/pants/backend/graph_info/tasks/dependees.py

```diff
@@ -35,11 +35,11 @@ def __init__(self, *args, **kwargs):
   def console_output(self, _):
     address_mapper = self.context.address_mapper
     build_graph = self.context.build_graph
-
     dependees_by_target = defaultdict(set)
+    addresses = address_mapper.scan_addresses()
```

Comment:
ah, I had forgot that the "special" v2 specs also work in the v1 path and was aiming for v1+v2 compatibility - but this is much cleaner. thx.